app/src/main/java/com/capstone/weatherapp/WeatherCache.kt

```diff
@@ -0,0 +1,75 @@
+package com.capstone.weatherapp
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class CityCache(
```

Comment:
We don't want or need the CityCache to store this much data. We only want to store the data we actually use. Look at the data you use to fill out the viewHolder for your City Picker screen. Only save that data to the db (humidity, min/max, temp, etc.). We don't need to save unnecessary stuff to the DB like `Clouds` that we never end up using.